app/views/lists/show.html.erb

```diff
@@ -1,40 +1,36 @@
 <% content_for(:title, @list.name) %>
-<div class="row">
-
-<h1><%= @list.name %> <span class="count">(<%= pluralize(@list.movies.size, "Movie") %>) <icon class="<%= public_private_indicator(@list) %>"><icon></span></h1>
-<p>by <%= @list.owner.username %></p>
 
-<% if @list.description.present? %>
-  <p><%= @list.description %></p>
-<% end %> <!-- #if description.present -->
+<div class="row">
+  <h1><%= @list.name %> <span class="count">(<%= pluralize(@list.movies.size, "Movie") %>) <icon class="<%= public_private_indicator(@list) %>"><icon></span></h1>
+  <p>by <%= @list.owner.username %></p>
 
+  <% if @list.description.present? %>
+    <p><%= @list.description %></p>
+  <% end %> <!-- #if description.present -->
 
   <%= form_tag user_list_path(@list.owner, @list), { class: "form-class", id: "list-show-sort", role: "form", method: :get } do %>
-    <%= select_tag :sort_by, options_for_select(Movie::LIST_SORT_OPTIONS, @sort_by), { class: "form-control index-sort-dropdown", :include_blank => "Sort by", id: "list_sort_options" } %>
+    <%= select_tag :sort_by, options_for_select(MovieDataService::LIST_SORT_OPTIONS, @sort_by), { class: "form-control index-sort-dropdown", :include_blank => "Sort by", id: "list_sort_options" } %>
```

Comment:
Aside from this line where I've updated the source of the `LIST_SORT_OPTIONS` constant, the rest of the changes in this file are just blankspace corrections.